publisher/lib/ontology_collector.py

```diff
@@ -1,106 +1,78 @@
 import argparse
-import os
+import os.path
+import re
+import sys
 
 from rdflib import Graph, OWL, RDF
 
+visited_ontologies = list()
+local_ontology_map_regex = re.compile('name="(.+)"\s+uri="(.+)"')
+local_ontology_map = dict()
+global output_graph
+
+
+def get_local_ontology_map(ontology_catalog_path: str):
+    try:
+        ontology_catalog_file = open(ontology_catalog_path, 'r')
+        ontology_catalog = ontology_catalog_file.read()
+        ontologies = local_ontology_map_regex.findall(string=ontology_catalog)
+        for ontology in ontologies:
+            local_ontology_map[ontology[0]]=ontology[1][2:]
+            v=local_ontology_map
+    except Exception as exception:
+        print(str(exception))
```

Comment:
Add an additional info about where in the code this exception was thrown, e.g. f"Exception occurred while getting local ontology map: {exception}"